internal/engine/protocol/activitypub/data_source.go

```diff
@@ -16,14 +21,15 @@ import (
 	"github.com/rss3-network/protocol-go/schema/network"
 	"github.com/samber/lo"
 	"github.com/sourcegraph/conc/pool"
-	"github.com/twmb/franz-go/pkg/kgo"
 	"go.uber.org/zap"
 )
 
 var _ engine.DataSource = (*dataSource)(nil)
-
 var DefaultStartTime int64
```

Comment:
DefaultStartTime is unexported but has a name suggesting it should be exported. Consider either exporting it or renaming to defaultStartTime

Reply:
Renamed it to `defaultStartTime` for it's only used in this file.